src/renderers/WebGLRenderer.js

```diff
@@ -2719,13 +2719,16 @@ function WebGLRenderer( parameters ) {
 
 	this.copyFramebufferToTexture = function ( position, texture, level ) {
 
-		var width = texture.image.width;
-		var height = texture.image.height;
+		level = level || 0;
```

Comment:
Is this the intent here?

```js
if ( level === undefined ) level = 0;
```

Reply:
Yes, following the resolved comment above.